src/utils/pixelSizeValidate.ts

```diff
@@ -0,0 +1,8 @@
+import config from 'config';
+
+export const pixelSizeValidate = (requestedPixelSize: number, sourcePixelSize: number): boolean => {
+  const resolutionTolerance = config.get<number>('resolutionTolerance');
```

Comment:
move to function attributes

Reply:
its arrow function, we talked about it before with other's PR's  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions